npm/src/getWeather.js

```diff
@@ -0,0 +1,33 @@
+const WEATHER_URL = `http://api.weatherstack.com/current?`;
+const WEATHER_KEY = 'access_key=0c1ddc3be04244cf802e7019a8f042bc';
+
+
+export const getWeather = (city, country) =>
+    fetch(`${WEATHER_URL}${WEATHER_KEY}&query=${country} ${city}`)
+    .then(response => response.json()).catch(function() {
+        alert('Smth went wrong');
+    })
+
+
+
+export const getDefaultWeather = () => fetch(`${WEATHER_URL}${WEATHER_KEY}&query=Minsk`)
+    .then(response => response.json());
+
+
+let getCoordinates = new Promise((resolve) => {
+    navigator.geolocation.getCurrentPosition((result) => {
+        let coordinates = {
+            latitude: result.coords.latitude,
+            longitude: result.coords.longitude
+        };
+        resolve(coordinates);
+    }, (error) => {
+        alert(`The location is not found.`);
+    })
+}).then(coordinates => { return coordinates });
+
+
+export const getWeatherforUserPosition = () => {
+    getCoordinates.then(coordinates => fetch(`${WEATHER_URL}${WEATHER_KEY}&query=${coordinates.latitude},${coordinates.longitude}`))
+        .then(res => res.json());
+}
```

Comment:
to make this work, you should check how you use it: 
```js
 let currentPosition = getWeatherforUserPosition();
    currentPosition.then(res => {
```

The code expects the function return a promise. In this case you should return it:
```suggestion
export const getWeatherforUserPosition = () => {
   return getCoordinates.then(coordinates => fetch(`${WEATHER_URL}${WEATHER_KEY}&query=${coordinates.latitude},${coordinates.longitude}`))
        .then(res => res.json());
}
```